client/src/panels/panel_declarations/results/result_components.js

```diff
@@ -360,7 +360,6 @@ const StatusIconTable = ({
               <span
                 className="link-unstyled sr-only"
                 aria-hidden="true"
-                tabIndex={0}
                 data-toggle="tooltip"
```

Comment:
Another case where it needs to take focus for the tooltip... but combining aria-hidden with a tabIndex is bad. If it remains focusable, it should at least not render in the text-only version

Reply:
Hmm actually, what is the purpose of this table anyways? This seems totally unnecessary

If this is for a11y, it should only be displayed in a11y mode, and also if anything, it should be incorporated into `VisibilityControl` component. But actually checking with screen reader, screen reader can't even read this properly because `aria-hidden="true"`. Not tab navigable either, and screen reader seems to be reading `VisibilityControl` component just fine. 

So I suggest that we remove this `<table>` entirely